src/Common/AzurePSCmdlet.cs

```diff
@@ -408,7 +408,7 @@ protected override void BeginProcessing()
         /// </summary>
         protected override void EndProcessing()
         {
-            if (CheckIfInteractive() && SurveyHelper.GetInstance().ShouldPropmtSurvey(this.MyInvocation.MyCommand.ModuleName, this.MyInvocation.MyCommand.Version))
+            if (SurveyHelper.GetInstance().ShouldPropmtSurvey(this.MyInvocation.MyCommand.ModuleName, this.MyInvocation.MyCommand.Version))
```

Comment:
@VeryEarly , could you please add one comment to function CheckIfInteractive() to describe the potential issue for some container environment?

Reply:
sure

Reply:
Shall we remove method CheckIfInteractive()?